pkg/controller/node/nodecontroller_test.go

```diff
@@ -326,6 +399,9 @@ func TestMonitorNodeStatusEvictPods(t *testing.T) {
 			evictionTimeout, util.NewFakeRateLimiter(), util.NewFakeRateLimiter(), testNodeMonitorGracePeriod,
 			testNodeStartupGracePeriod, testNodeMonitorPeriod, nil, false)
 		nodeController.now = func() unversioned.Time { return fakeNow }
+		for _, ds := range item.daemonSets {
```

Comment:
Can you populate the testclient.Fake (that's embedded in the FakeNodeHandler) with the DaemonSet, so that "ds" is automatically populated into the nodeController.daemonSetStore the way nodes and pods get populated into the nodeController.{nodeStore,podStore} via the testclient.Fake? If you do it I think
https://github.com/mml/kubernetes/blob/forgive-me-not/pkg/controller/node/nodecontroller.go#L210
should return the DaemonSet. Inserting it manually seems a little hacky. (If it's too hard to do it the way I described then I'm fine with what you did but it would be good to try to do it the same way the other objects are handled.)